packages/core/src/resolver.js

```diff
@@ -102,15 +113,27 @@ export default class Resolver {
     throw new Error(`Cannot resolve "${path}" from "${resolvedPath}"`)
   }
 
-  requireModule(path, { esm, alias, intropDefault } = {}) {
+  requireModule(path, { esm, isEsm = esm, alias, isAlias = alias, intropDefault, shouldIntropDefault = intropDefault } = {}) {
```

Comment:
isESm -> useESM? 

Also shouldIntropDefault looks little verbose to me for usage in code :)